### DISCUSSION TITLE
Not Connected error

### DISCUSSION THREAD

**Discussion by mikebeeboy** [apohl79/audiogridder#988] [Help] [answered]
### Discussed in https://github.com/apohl79/audiogridder/discussions/888

<div type='discussions-op-text'>

<sup>Originally posted by **middleshelf** January 19, 2022</sup>
Hi,
New user here, so apologies if this is super obvious. I looked through the documentation and can't seem to find the answer. Hopefully someone can help here.

I installed the Audiogridder server on my Windows machine and the client on my Mac. I'm using the stable release version 1.1.1 (stable) and am able to load an AGridder instrument into Logic with no issues and scan all my plugins on the Windows machine with no issues either. 

However the status shows "not connected" when I load the instrument into Logic (same happens with FX)
![image](https://user-images.githubusercontent.com/1553098/150275608-01c05c1a-aed0-4f8f-bb40-78def564cffc.png)

I am hard wired to the same router:
my server IPv4 Addy  (Windows) is 192.168.51.70
and my client IPv4 Addy (Mac) is 192.168.51.236

I can ping my Mac IP from Windows with no issue, but when I ping the windows machine from the Mac - I get no response. 

Thanks for your help.

-Walt</div>

**Comment by mikebeeboy** [comment]
I have same setup and can ping both ways but audiogridder makes a loud noise and wont stop loading!

**Comment by mikebeeboy** [comment]
could it be because my mac is a hackentosh?

**Comment by apohl79** [comment, marked as answer]
Have you checked the windows firewall? Try deactivating it temporarily to see if this solves the issue.

**Reply by mikebeeboy** [reply]
Thank you bro! I dont know how I missed such a simple thing by purposely dismissing it as a cause!
